pkg/policies/opa/rego/docker/docker_run/AC_DOCKER_0052.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "runusingupgrade",
+    "file": "runusingupgrade.rego",
+    "policy_type": "docker",
+    "resource_type": "run",
+    "template_args": {
+        "prefix": "",
+        "suffix": "",
+        "name": "runusingupgrade"
+    },
+    "severity": "HIGH",
+    "description": "Ensure that use of Commands 'apt-get upgrade' and 'apt-get dist-upgrade' should not be used",
```

Comment:
Incorrect grammar, rule content must be high quality. Please go through existing policies and think few times before writing rule content. Content is as much important as policy code.